website/source/partials/helper/communicator/_SSH-not-required.html.md

```diff
@@ -79,10 +79,10 @@
     
 -   `ssh_proxy_password` (string) - The optional password to use to authenticate with the proxy server.
     
--   `ssh_keep_alive_interval` (time.Duration) - How often to send "keep alive" messages to the server. Set to a negative
+-   `ssh_keep_alive_interval` (duration string | ex: "1h5m2s") - How often to send "keep alive" messages to the server. Set to a negative
     value (`-1s`) to disable. Example value: `10s`. Defaults to `5s`.
     
--   `ssh_read_write_timeout` (time.Duration) - The amount of time to wait for a remote command to end. This might be
+-   `ssh_read_write_timeout` (duration string | ex: "1h5m2s") - The amount of time to wait for a remote command to end. This might be
```

Comment:
would this be a correct way to document this more obviously ? Somehow netlify is not deploying previews anymore but we can imagine.